### ISSUE TITLE
Enums bugs

### ISSUE THREAD

**Issue by d0d04m3** [kaitai-io/kaitai_struct_webide#139]
so i use https://ide.kaitai.io/
and use this file
`samples/sample1.zip`
and this my code

```ksy
meta:
  id: animal_record
  endian: be
seq:
    - id: frametype
      type: u1
      enum: frame
enums:
    frame:
      0x10: fixed
      0x68: variable
```

and this my resualts 
`frametype = FIXED|VARIABLE (0x50 = 80)`
how that its worng

**Comment by generalmimon** [comment]
> Reading the webide code gave me come clues what us going on. As i understand this flags behavior is webide specific and is not part of generated code?

Yes, this is a misfeature of the Web IDE that doesn't respect how KS enums work. Currently, KS enums are strictly one-to-one bi-directional mappings between integer constants and identifiers (one integer value can be assigned to at most one identifier and one identifier can't be used more than once). The presence of an enum member is always based on equality - so an integer converted to an enum type matches at most one member in the enum.

**Comment by wader** [comment]
@generalmimon Good to know. It confused me a bit as my case it only showed one enum name (no "|") so thought i had missed something how enums work.
